R/stat-function.r

```diff
@@ -5,7 +5,7 @@
 #' the x axis, and the results are drawn (by default) with a line.
 #'
 #' @eval rd_aesthetics("stat", "function")
-#' @param fun function to use. Must be vectorised.
+#' @param fun function to use or formula. Must be vectorised.
```

Comment:
Please expand this sentence a bit more. Now that we have two different possible inputs, the documentation needs a little more care. Also, please add a simple example to the examples, or maybe better rewrite one or two of the current ones (so we don't increase the total number of examples).

Reply:
The `.x` in the formula should also be explained here.

Reply:
And we also need a link to `as_function`'s doc.

Reply:
Happy to update the docs for the argument and update the examples more fully, if you'd like, I was just trying to be as minimalist as possible. Want me to use `.` or `.x` for the formula stuff? (@yutannihilation brought the `.` vs `.x` part up in another comment, too.)